dask_geomodeling/utils.py

```diff
@@ -69,12 +69,22 @@ def get_dtype_min(dtype):
     return np.iinfo(d).min
 
 
+def get_int_dtype(n):
+    """Get the smallest int dtype that accomodates 'n' values"""
+    for dtype in ("i1", "i2", "i4", "i8"):
```

Comment:
What is wrong with `f"d{2 ** ceil(log((1 + log(max(n, 1), 2) // 8), 2))}"`? Kidding! But I want to let you know that that is what brings fun to programming for me... It fails on the 2 ** 64 - 1 case, because log(n, 2) == 64.0 somehow. Puzzling. But I will not pursue it further.

Reply:
Leave the one liners to Arjan :)